packages/react-native/ReactCommon/react/renderer/animations/LayoutAnimationKeyFrameManager.cpp

```diff
@@ -1118,7 +1118,10 @@ ShadowView LayoutAnimationKeyFrameManager::createInterpolatedShadowView(
       propsParserContext,
       progress,
       startingView.props,
-      finalView.props);
+      finalView.props,
+      finalView.layoutMetrics.frame.size.width,
+      finalView.layoutMetrics.frame.size.height
```

Comment:
I double check that the spec defaults to `border-box` for reference box by default, so this should be good for now.